cmd/run.go

```diff
@@ -67,6 +68,11 @@ A full workflow is capable of running the following steps:
 			Shorthand:   "b",
 			Description: "the subdirectory of the repository where the SDK is located in the repo, helps with documentation generation",
 		},
+		model.BooleanFlag{
+			Name:        "published",
+			Shorthand:   "p",
+			Description: "whether the SDK is published to a package manager or not, determines the type of installation instructions to generate",
+		},
```

Comment:
Needed for action

Reply:
Is this staying as an argument in the action?

Reply:
Think we need it until publishing gets supported by `run`